templates/tree.html.erb

```diff
@@ -38,8 +38,14 @@
 </style>
 <body>
   <div id="tree-container"></div>
+<% if options[:local_dependencies]%>
+  <% Dir.entries("js").each do |jsfile|%>
+    <%= "<script>#{File.read File.join('js',jsfile)}</script>" if jsfile != '.' && jsfile != '..' %>
+  <%end%>
+<% else%>
```

Comment:
that is an interesting approach, but there are some points we need to take into consideration before merging that code.
1- there is no need to store the javascript files in our project, it could be loaded from the internet, that makes the gem cleaner
2- we don't have to change the template itself, the template uses the script tag with `src` attribute, and dirtree executable should replace any of these script tags on the fly with the content if `local_dependencies` exists.